[ready for publication] Spanish - Introduction to Web Accessibility

rumoroso opened the pull request.

Commit:
add Spanish translation (first iteration)

shawna-slh committed:
Update index.es.md

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
```
Should we translate `title=Summary"` into `title="Resumen"`?

I am translating it manually, but I wonder whether this will be automated or not, because this needs to be done in all the documents.

shawna-slh replied:
I created [this GitHub issue](https://github.com/w3c/translation-glossaries/issues/3) for you all to agree on translation of "Summary" for these boxes. :-)

You will need to keep doing it manually for now. Maybe eventually we'll automate that, yet not any time soon.

Thanks.

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+<li><a href="#what">Qué es la Accesibilidad Web</a></li>
+<li><a href="#important">La Accesibilidad es Importante para las Personas, Empresas y Sociedad</a></li>
+<li><a href="#making">Hacer la Web Accesible</a></li>
+<li><a href="#evaluate">Evaluar Accesibilidad</a></li>
```
I would use "Evaluar la Accesibilidad", or even better "Evaluando la Accesibilidad". Anyway, the article ("la") is missing there.

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+  <footer><cite>Tim Berners-Lee, Director del W3C Director e inventor de la World Wide Web</cite></footer>
+</blockquote>
+
+La Web está fundamentalmente diseñada para trabajar para todo el mundo, independientemente del hardware, software, idioma, ubicación o capacidad. Cuando la Web cumple ese objetivo, es accesible para personas con un diverso rango de audición, movimiento, visión y habilidades cognitivas.
```
A proposal to avoid repeating the word "para":

"La Web está fundamentalmente diseñada para que todo el mundo pueda usarla […]"

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+
+La Web está fundamentalmente diseñada para trabajar para todo el mundo, independientemente del hardware, software, idioma, ubicación o capacidad. Cuando la Web cumple ese objetivo, es accesible para personas con un diverso rango de audición, movimiento, visión y habilidades cognitivas.
+
+Por lo tanto, el impacto de la discapacidad cambia radicalmente en la Web porque elimina las barreras de comunicación e interacción que muchas personas enfrentan en el mundo físico. Sin embargo, cuando los sitios web, aplicaciones, tecnologías o herramientas están mal diseñados, pueden crear barreras que excluyen a las personas del uso de la Web.
```
Proposal:

"el impacto de la discapacidad cambia radicalmente en la Web porque **ésta** elimina las barreras de comunicación e interacción que muchas personas **encuentran** en el mundo físico"

rumoroso replied:
I agree with "encuentran" (much better than what I wrote initially), but I would avoid adding "ésta" because "elimina" has implied subject :/

cmuncharaz replied:
Yep, but it looks a bit ambiguous if the subject is left implied. The subject in the sentence is "el impacto de la discapacidad", so if it's not clear that in the subordinate clause the subject is "la Web", one might imply that the subject is "el impacto de la discapacidad". Don't you think so?

Notice how the original text in English makes clear what the subject is ("because the Web removes barriers […]").

rumoroso replied:
ok, you convinced me ;) and I prefer not to overthink of it (which btw I do quite often) :)
Thanks for the review Carlos

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+
+Accesibilidad Web significa que sitios web, herramientas y tecnologías están diseñadas y desarrolladas de tal manera que las personas con discapacidades pueden usarlas. Más concretamente, las personas pueden:
+
+-   percibir, comprender, navegas e interactuar con la Web
```
"navegas" should be **"navegar"**.

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+{% include box.html type="end" %}
+{:/}
+
+## La Accesibilidad es Importante para las Personas, Empresas y Sociedad {#important}
```
I am leaving this comment here, but it could be applied anywhere else where appropriate.

In the Spanish language, the capital letters are not used in every word belonging to a heading. This is done in the English language, but the Spanish grammar states that in headings, titles, works of art, etc... only the first word should be capitalized (check rule 4.17 in the link below).

More info:
http://lema.rae.es/dpd/srv/search?id=BapzSnotjD6n0vZiTp
https://cvc.cervantes.es/lengua/alhabla/museo_horrores/museo_011.htm

shawna-slh replied:
I added this to the [General glossary](https://github.com/w3c/translation-glossaries/blob/master/general.md#capitalization---titles-and-headings) and to the [Spanish glossary](https://github.com/w3c/translation-glossaries/blob/master/Castellano-Spanish.md#capitalization---title-and-headings).

Feel free to improve the wording there if you would like to.

Also, **please do add other things like this to the Glossaries**.

<hr>

@rumoroso I guess you want to update https://www.w3.org/WAI/standards-guidelines/mobile/es
:-)

cmuncharaz replied:
Great, thanks @slhenry! Will make sure to add comments like this one to the Glossary next time.

cmuncharaz commented on index.es.md:
```diff
@@ -0,0 +1,244 @@
+
+Al desarrollar o rediseñar un sitio web, evalúe la accesibilidad desde el principio y durante todo el proceso de desarrollo para identificar los problemas de accesibilidad desde el inicio, cuando es más fácil resolverlos. Pasos sencillos, como cambiar la configuración del navegador, pueden ayudarle a evaluar algunos aspectos de la accesibilidad. Una evaluación exhaustiva para determinar si un sit ...
+
+Existen herramientas que ayudan en con la evaluación. Sin embargo, ninguna herramienta por sí sola puede determinar si un sitio cumple con las pautas de accesibilidad. Se requiere una evaluación humana bien experimentada para determinar si un sitio es accesible.
```
"Existen herramientas que ayudan **en con** la evaluación". We should get rid of "en" or "con", only one preposition should be used.

Commit:
fix errors and improvements after first review

Commit:
Merge remote-tracking branch 'origin/spanish-translation' into spanish-translation

# Conflicts:
#	index.es.md

rumoroso committed:
Update index.es.md

Update sentence by adding the subject explicitly

rumoroso commented:
Thanks for the review Carlos. With the changes I also added your name to the contributors list.
cc: @cmunch

cmuncharaz commented:
> Thanks for the review Carlos. With the changes I also added your name to the contributors list.
> cc: @cmunch

Awesome, great job!

shawna-slh committed:
sync updates to English version

shawna-slh commented on index.es.md:
```diff
@@ -0,0 +1,246 @@
+
+### Entrada con teclado
+
+![mouse crossed out](https://www.w3.org/WAI/intro/no-mouse.png){:.left width="67" height="45"}
```
"mouse crossed out"

needs translation

cmuncharaz replied:
"Mouse crossed out" could be translated as "Ratón tachado con una cruz"

shawna-slh commented on index.es.md:
```diff
@@ -0,0 +1,246 @@
+
+### Alternativas textuales para imágenes
+
+![image of logo; HTML markup img alt='Iniciativa de Accesibilidad Web'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}
```
This is alternative text for [this image](https://www.w3.org/WAI/intro/alt-logo.png). For at least a while, we will not change the text in the image (we might sometime later when we get to make it SVG , but it's not a priority for now). Therefore, I think should translate:

"image of logo" and "markup"

and leave in English "Web Accessibility Initiative logo"

cmuncharaz replied:
The text "image of logo; HTML markup img alt='Iniciativa de Accesibilidad Web'" could be translated as "Imagen de logotipo; Marcado HTML img alt='Web Accessibility Initiative logo'".

shawna-slh committed:
translations per https://github.com/w3c/wai-intro-accessibility/pull/42#pullrequestreview-261946368

Thanks @cmuncharaz !

@rumoroso , if you suggest different translation, we can change it. I wanted to go ahead and get something now so we can publish it. :-)
